gbpservice/nfp/configurator/agents/loadbalancer_v1.py

```diff
@@ -57,6 +57,10 @@ def update_status(self, obj_type, obj_id, status, agent_info, obj=None):
         :param obj_type: object type
         :param obj_id: object id
         :param status: status of the object to be set
+        :param agent_info: agent_info
```

Comment:
Can the comments be more verbose. Need not provide the contents of these variables. Specify the purpose of this parameter.